### PR TITLE
Gitflow release model & Architectural Decision Records adopted 

### PR DESCRIPTION
**Pull request by ashepherd**
This pull request adopts a Gitflow style for creating `feature` branches, which are merged into a `develop `branch. Releases occur by pulling `develop` branch into `master`.

Architectural Decision Records (ADR) are markdown files that summarize complex and lengthy discussions within the Issue queue, telecons and face-to-face meetings. ADRs will be placed in the `/decisions` directory.

### CODE REVIEW

File: decisions/13-schemaorg-identifier-as-PropertyValue.md [hunk 1]
```diff
@@ -1,5 +1,7 @@
 # schema.org/identifier as schema.org/PropertyValue over Text/URL #
```
Comment (rduerr) [inline decisions/13-schemaorg-identifier-as-PropertyValue.md]: See comments on issue #13 - further specification here is needed.

**Comment by ashepherd** [comment]
@rduerr , are you ok if we approve this workflow now, and treat the identifier issue #13 inn a separate pull? I think all @mbjones did here was rename the existing ADR about the identifier type. Your points are well taken and should be addressed. Just wasn't sure if we should block this pull request for it as it helps us roll out individual fixes in na clearer way to the community. In fact, #13 could be our first demonstration of that.

File: decisions/30-release-workflow.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+## Status ##
+_Proposed_
+
+## Decision ##
+
+We will use a [GitFlow](https://nvie.com/posts/a-successful-git-branching-model/)-inspired release model to track proposed changes, their implementation, and the associated Architectural Decision Record.
+
+## Context  -- What is the issue that we're seeing that is motivating this decision or change.
+Because people will review our guidance documents online, it will be helpful to be explicit about versioning and releases of the guidance and other documents.  In particular, in GitHub, users will normally see the `master` branch, which therefore should reflect the current stable release of the documentation (rather thanb confusing people with in-progress proposed changes that are not yet released).
```
Comment (datadavev) [inline decisions/30-release-workflow.md]: ... (rather than~~b~~ confusing ...

File: decisions/30-release-workflow.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+
+Consequently, we will use a [GitFlow](https://nvie.com/posts/a-successful-git-branching-model/)-inspired release model in which the `master` branch always reflects the current stable release, a `develop` branch is used for merging finished proposals being prepared for release, and `feature` branches are used for creating changes to implement specific proposals that are reflected in an Architectural Decision Record.  For changes that do not require a formal decision via and ADR, such as spelling corrections, grammatical rewording, etc., maintainers can commit changes directly to the `develop` branch, and other contributors can do a pull request directly against the `develop` branch.  The use of feature branches is really focused on managing proposals that need discussion, review, and a dec ision through an ADR. Maintainers will make judgement calls on whether an ADR is needed, and might convert contributed pull requests to a feature branch if they determine that an ADR is needed.
```
Comment (datadavev) [inline decisions/30-release-workflow.md]: ... decision via an~~d~~ ADR ...

... and a ~~dec ision~~ decision through an ADR. ...

File: decisions/13-schemaorg-identifier-as-PropertyValue.md [hunk 4]
```diff
@@ -58,4 +60,4 @@ is the same as
 }
```
Comment (datadavev) [inline decisions/13-schemaorg-identifier-as-PropertyValue.md]: In the third example, is the intent really to assert that the graph node of type `identifier` has `id` of `"https://doi.org/10.1234/56789"`? Seems that `id` value should really be on the Dataset class.

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+
+- Creating and discussing proposals through [GitHub issues](https://github.com/ESIPFed/science-on-schema.org/issues)
+- Implementing proposed features by editing the guidelines and extension documentation, and editing the Architectural Decision Record for the proposal
+- Issuing a Pull Request to request review of the proposed changes, and to discuss the feature and iterate on development
+
+# Development and Release process
+Because people will review our guidance documents online, it will be helpful to be explicit about versioning and releases of the guidance and other documents.  In particular, in GitHub, users will normally see the `master` branch, which therefore should reflect the current stable release of the documentation (rather thanb confusing people with in-progress proposed changes that are not yet released).
```
[the remark this answers is not in the document]
Comment (mbjones) [reply CONTRIBUTING.md]: Fixed in sha 4c10362

File: CONTRIBUTING.md [hunk 6]
```diff
@@ -0,0 +1,33 @@
...
+
+Consequently, we will use a [GitFlow](https://nvie.com/posts/a-successful-git-branching-model/)-inspired release model in which the `master` branch always reflects the current stable release, a `develop` branch is used for merging finished proposals being prepared for release, and `feature` branches are used for creating changes to implement specific proposals that are reflected in an Architectural Decision Record (ADR).  For changes that do not require a formal decision via and ADR, such as spelling corrections, grammatical rewording, etc., maintainers can commit changes directly to the `develop` branch, and other contributors can do a pull request directly against the `develop` branch.  The use of feature branches is really focused on managing proposals that need discussion, review, and a decision through an ADR. Maintainers will make judgement calls on whether an ADR is needed, and might convert contributed pull requests to a feature branch if they determine that an ADR is needed.
```
[the remark this answers is not in the document]
Comment (mbjones) [reply CONTRIBUTING.md]: fixed in sha 4c10362

File: decisions/30-release-workflow.md (hunk 2 above)
Comment (mbjones) [reply decisions/30-release-workflow.md]: fixed in sha 4c10362

File: decisions/30-release-workflow.md (hunk 3 above)
Comment (mbjones) [reply decisions/30-release-workflow.md]: fixed in sha 4c10362

File: decisions/13-schemaorg-identifier-as-PropertyValue.md (hunk 4 above)
Comment (mbjones) [reply decisions/13-schemaorg-identifier-as-PropertyValue.md]: While I renamed the decision file for `13-schemaorg-identifier-as-PropertyValue.md` in this PR, I think we should move substantive discussion of its content to another issue or PR. Probably in issue #13.

File: decisions/13-schemaorg-identifier-as-PropertyValue.md (hunk 1 above)
Comment (mbjones) [reply decisions/13-schemaorg-identifier-as-PropertyValue.md]: Agreed we need to discuss that, but let's do so over in issue #13.  All I did in this PR was to rename the ADR file to match the convention proposed in the new workflow.  So I'm going to resolve this comment.

File: decisions/13-schemaorg-identifier-as-PropertyValue.md (hunk 4 above)
Comment (mbjones) [reply decisions/13-schemaorg-identifier-as-PropertyValue.md]: Copied comment to issue #13

**Comment by rduerr** [comment]
@ashepherd I am fine with moving issue #13 to a different pull request and getting the process change rolled out.

**Pull request merged**